mkdocs.yml

```diff
@@ -0,0 +1,48 @@
+site_name: Backfila
+repo_name: Backfila
+repo_url: https://github.com/cashapp/backfila
+site_description: "Backfila: Service that manages backfill state, calling into other services to do batched work"
+site_author: Square, Inc.
+remote_branch: gh-pages
+
+copyright: 'Copyright &copy; 2019 Square, Inc.'
+
+theme:
+  name: 'material'
+  favicon: img/favicon.ico
+  logo: img/cashapp.png
+  feature:
+    tabs: false
+  palette:
+    primary: 'grey'
+    accent: 'red'
+
+extra_css:
+  - 'css/app.css'
+
+markdown_extensions:
+  - smarty
+  - codehilite:
+      guess_lang: false
+  - footnotes
+  - meta
+  - toc:
+      permalink: true
+  - pymdownx.betterem:
+      smart_enable: all
+  - pymdownx.caret
+  - pymdownx.inlinehilite
+  - pymdownx.magiclink
+  - pymdownx.smartsymbols
+  - pymdownx.superfences
+  - tables
+
+nav:
+  - 'Overview': index.md
+  - '0.x API':
+      - 'client': 0.x/client/index.md
+      - 'client-service': 0.x/client-service/index.md
```

Comment:
Are these all right?

Reply:
Yes, they match the Gradle subproject names (which could be changed later to make it clearer ie. backfila-client, backfila-client-service, backfila-service)

Reply:
ACK